src/ansys/mapdl/core/mapdl.py

```diff
@@ -3204,11 +3204,31 @@ def run(
         if "\n" in command or "\r" in command:
             raise ValueError("Use ``input_strings`` for multi-line commands")
 
-        # check if we want to avoid the current non-interactive context.
+        # Check kwargs
+        verbose = kwargs.pop("verbose", False)
+        save_fig = kwargs.pop("savefig", False)
+
+        # Check if you want to avoid the current non-interactive context.
         avoid_non_interactive = kwargs.pop("avoid_non_interactive", False)
 
+        # Check if there is an unused keyword argument. If there is, it
+        # might be because you wrote a wrong argument name.
+        #
+        # Remove empty string kwargs
+        for key, value in list(kwargs.items()):
```

Comment:
```suggestion
        for key, value in kwargs.items():
```
I don't think the `list` is useful

Reply:
If you don't use list,  you will be removing elements of the dict in the loop, which might decrease its size and giving later error.

Reply:
You are right